src/sentry/static/sentry/app/views/settings/components/dataPrivacyRulesPanel/dataPrivacyRulesPanel.tsx

```diff
@@ -237,24 +237,21 @@ class DataPrivacyRulesPanel extends React.Component<Props, State> {
   };
 
   render() {
-    const {panelHeaderSubTitle, disabled} = this.props;
+    const {additionalContext, disabled} = this.props;
     const {rules, savedRules} = this.state;
     const hideButtonBar = savedRules.length === 0 && rules.length === 0;
     return (
       <React.Fragment>
         <Panel>
-          <StyledPanelHeader>
-            {t('Data Privacy Rules')}
-            {panelHeaderSubTitle && (
-              <PanelHeaderSubTitle>{panelHeaderSubTitle}</PanelHeaderSubTitle>
-            )}
-          </StyledPanelHeader>
-          <PanelAlert type="info" icon="icon-docs">
+          <StyledPanelHeader>{t('Data Privacy Rules')}</StyledPanelHeader>
+          <PanelAlert type="info">
+            {additionalContext}
+            {t('To learn more about datascubbing, ')}
```

Comment:
We likely want to use `tct` here to allow translators to see the full text (we already use this further down in the diff), not sentence fragments.

Reply:
Sounds good - I'll update this :)